### PR TITLE
updated v3.8.15

### PR DESCRIPTION
**Pull request by mbargull**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mbargull** [comment]
@conda-forge-admin, please rerender

**Comment by mbargull** [comment]
@conda-forge-admin, please rerender

File: .github/workflows/automerge.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
         uses: actions/checkout@v3
       - name: automerge-action
         id: automerge-action
-        uses: conda-forge/automerge-action@main
+        uses: conda-forge/automerge-action@master
```
Comment (mbargull) [inline .github/workflows/automerge.yml:16]: It doesn't really matter, but interestingly enough, rendering yields  `@master` for the `3.8` and `3.9` branches and `@main` for the `3.10` branch.
(I was a bit confused when I noticed that with my first local rerendering, hence querying the bot multiple times to get confirmation it really wants that change.)

**Comment by mbargull** [comment]
@conda-forge-admin, please restart ci

File: .github/workflows/automerge.yml (hunk 1 above)
Comment (bollwyvl) [reply .github/workflows/automerge.yml:16]: Looks like a `conda-forge.yml` change would do it:

https://github.com/conda-forge/python-feedstock/blob/main/conda-forge.yml#L12-L14

vs

https://github.com/conda-forge/python-feedstock/blob/3.8/conda-forge.yml

Might be worth syncing up some of the other stuff, as well...

Comment (mbargull) [reply .github/workflows/automerge.yml]: Thanks for digging, @bollwyvl!

**Comment by jakirkham** [comment]
Thanks Marcel! 🙏

**Comment by h-vetinari** [comment]
Hey @mbargull @jakirkham 

The merge of this PR to the `3.8` branch failed to build for one PPC version (unfortunately, the more relevant OpenSSL 3 one), and I'm now seeing resolver errors due this. Would you be so kind to restart it please?

**Comment by jakirkham** [comment]
Tried [restarting CI]( https://app.travis-ci.com/github/conda-forge/python-feedstock/builds/260046627 ). Though we may need a re-render of this branch

**Comment by h-vetinari** [comment]
I think this is now running into the setuptools 66 stuff in the `downstreams:` tests...
```
TEST START: /home/conda/feedstock_root/build_artifacts/pkg_cache/setuptools-66.1.1-pyhd8ed1ab_0.conda

Traceback (most recent call last):
[...]
RuntimeError: Could not read JSON repodata file (/home/conda/feedstock_root/build_artifacts/pkg_cache/cache/51e90255.json) parse error line 90: subdir mismatch
```

**Comment by jakirkham** [comment]
Could you please send a PR to either constrain this to `setuptools <66.0.0a` or if necessary skip this test altogether?

**Comment by h-vetinari** [comment]
> Could you please send a PR to either constrain this to `setuptools <66.0.0a` or if necessary skip this test altogether?

I don't think `downstreams:` can work with a constraint? Perhaps best to skip it for now?

**Comment by isuruf** [comment]
> I think this is now running into the setuptools 66 stuff in the downstreams: tests...

No. This should have been fixed by https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/388. Please revert your changes.

**Comment by h-vetinari** [comment]
> This should have been fixed by [conda-forge/conda-forge-repodata-patches-feedstock#388](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/388).

But it wasn't fixed by that. John restarted the travis build ~5h ago, the PR you linked was merged >24h ago.

>  Please revert your changes.

There's not much to revert (especially considering how `aarch64` was skipped due to long-obsolete constraints on drone), but in any case: https://github.com/conda-forge/python-feedstock/pull/619

**Comment by mbargull** [comment]
Downstream test issues are non-functional due to https://github.com/conda/conda-build/issues/4750 (i.e., an issue we experience through `conda-mambabuild` caused by a `conda-build`+`libsolv=0.7.23` incompatibility).

**Comment by isuruf** [comment]
Sorry, wrong PR. Correct PR is https://github.com/conda-forge/boa-feedstock/pull/61

**Pull request merged**